intermol/lammps/lammps_parser.py

```diff
@@ -534,9 +536,10 @@ def parse_pair_modify(self, line):
             elif line[2] == 'arithmetic':
                 self.system.combination_rule = 'Lorentz-Berthelot'
             else:
-                raise ValueError('Unsupported pair_modify mix argument in input file!')
+                raise LammpsError('pair_modify mix argument "{}" is not yet supported'
+                                  'in InterMol'.format(line[2]))
         else:
-            raise ValueError('Unsupported pair_modify style in input file!')
+            raise LammpsError('pair_modify style not yet supported in InterMol')
```

Comment:
Hmm right now I've only added `UnimplementedConversion` which is more specific for functional forms. I guess I should generalize it a bit more to work for any unimplemented feature.